packages/react/src/component-tests/IcPageHeader/IcPageHeader.cy.tsx

```diff
@@ -0,0 +1,282 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBreadcrumb,
+  IcBreadcrumbGroup,
+  IcButton,
+  IcChip,
+  IcNavigationItem,
+  IcPageHeader,
+  IcStep,
+  IcStepper,
+  IcTextField,
+} from "../../components";
+import { mount } from "cypress/react";
+import { BE_VISIBLE, HAVE_LENGTH } from "../utils/constants";
+import {
+  DesktopScrollablePageVariant,
+  ScrollStickyPageHeader,
+} from "./IcPageHeaderTestData";
+
+const DEFAULT_TEST_THRESHOLD = 0.05;
+
+describe("IcPageHeader e2e, A11y and visual regression tests", () => {
```

Comment:
Can this be updated to just `IcPageHeader` and the `it(` further down detail what the tests are. For example, the first test is just labelled `IcPageHeader`